src/app/actions/iteration.actions.ts

```diff
@@ -35,20 +43,17 @@ export class GetError implements Action {
 }
 
 export class AddSuccess implements Action {
+  payload : any;
```

Comment:
Why is it named payload when in other Actions it's named as iteration? and also the type shouldn't be ```any```.